zap/src/main/java/org/zaproxy/zap/extension/api/WebUI.java

```diff
@@ -479,7 +479,7 @@ public String handleRequest(URI uri, boolean apiEnabled) {
                                 + API.API_NONCE_PARAM
                                 + "="
                                 + API.getInstance()
-                                        .getLongLivedNonce("/OTHER/core/other/proxy.pac/")));
+                                        .getLongLivedNonce("/OTHER/network/other/proxy.pac/")));
```

Comment:
Should we link to the shortcut path instead?

Reply:
Either way better to extract the path from the resource message and have it in a single place. (Same for the other API endpoint.)